src/main/java/org/broadinstitute/hellbender/tools/exome/acnvconversion/ACNVModeledSegmentConversionUtils.java

```diff
@@ -27,9 +27,23 @@ private ACNVModeledSegmentConversionUtils() {}
         return acnvModeledSegment.stream().map(s -> convertACNVModeledSegmentToModeledSegment(s, genome)).collect(Collectors.toList());
     }
 
+
     private static ModeledSegment convertACNVModeledSegmentToModeledSegment(final ACNVModeledSegment acnvModeledSegment, final Genome genome) {
+
+        final TargetCollection<TargetCoverage> targets = genome.getTargets();
+        return convertACNVModeledSegmentToModeledSegment(acnvModeledSegment, targets);
+    }
+
+    @VisibleForTesting
+    static ModeledSegment convertACNVModeledSegmentToModeledSegment(ACNVModeledSegment acnvModeledSegment, TargetCollection<TargetCoverage> targets) {
+
+        // Make sure that we do not let segment mean become zero
+        double updatedCenter = acnvModeledSegment.getSegmentMeanPosteriorSummary().center();
+        if (Math.pow(2, updatedCenter) <= 0) {
+            updatedCenter = ParamUtils.log2(TangentNormalizer.EPSILON);
+        }
+
         return new ModeledSegment(acnvModeledSegment.getInterval(), ModeledSegment.NO_CALL,
-                genome.getTargets().targetCount(acnvModeledSegment.getInterval()),
-                acnvModeledSegment.getSegmentMeanPosteriorSummary().center());
+                targets.targetCount(acnvModeledSegment.getInterval()), updatedCenter);
```

Comment:
Never mind previous comments, I see that `ACNVModeledSegment` starts out in log space.